libbluetoe.cpp

```diff
@@ -0,0 +1,10 @@
+/*
```

Comment:
Well, yes a hundred years ago, when source code tools like git weren't available, this kind of information might have made sense. But today, we have all of this in git :-)